examples/s3-import/provider.tf

```diff
@@ -0,0 +1,8 @@
+provider "aws" {
+  region = var.region_1
```

Comment:
See my comment on the variables.tf of this example asking to consider renaming these two variables